### DISCUSSION TITLE
Download collection 2

### DISCUSSION THREAD

**Discussion by ffilipponi** [yannforget/landsatxplore#28] [Ideas]
Thanks for nice tool, very useful and friendly, to download LANDSAT data.
I am wondering if the integration of LANDSAT collection 2 datasets (both level 1 and level 2) is planned among future code improvements.
Thanks for taking time to read this new feature proposal.

Federico Filipponi

**Comment by yannforget** [comment]
Hi Federico and thanks!

It is planned but I haven't started to work on it. It's gonna take some time as collection 2 relies on a different API and download procedure (an order is necessary before download). I don't known when I will have the time to implement it...
Best regards
Yann

**Comment by lukecarberry** [comment]
Hi Yann,

I really love this tool, thank you for spending so much effort to make it! Did you ever set up landsatxplore to download collection 2 files? I am trying to download them but having trouble currently. I'm getting: `landsatxplore.errors.EarthExplorerError: Download is not available` Thanks! 

Luke
